utils/README.md

```diff
@@ -1,15 +1,21 @@
-# BUFR Utilities
 
-This is a collection of commonly used BUFR utilities:
-[readbp](#readbp) - Read prepbufr file and print reports
-[readmp](#readmp) - Read any BUFR file and dump contents of each subset one at a time
-[binv](#binv) - Print BUFR file inventory by message type
-[sinv](#sinv) - Print inventory of satellite data by platform and instrument type
-[cmpbqm](#cmpbqm) - Print inventory of prepbufr observations by variable, report type and quality marks
+#### Sample Utilities
+@brief Collection of commonly-used utilities based on the library
```

Comment:
the `@brief` shows up on page.

Reply:
Yeah, I know but we need it for Doxygen, and I figured it didn't detract too much from the overall readability.  As I noted above, Doxygen and GitHub have different rules about what markdown extensions they will and won't accept, so in some places I had to compromise a little to make it work in both places.

Reply:
Yeah. I don't like that.  One has to click to go to doxygen page or build it.  This is the front page